src/components/evaluation/charts/BarChart.js

```diff
@@ -49,6 +49,25 @@ const calculatePercentage = (questionType, count, totalResponses) => {
   return ''
 }
 
+// break point for too small bars
+// if the percentual responses of a bar are smaller than the given
+// value, the label (A, B, ...)  is not displayed on top of the bar
+const smallBarBreak = 0.05
```

Comment:
Should be defined as a constant in the constants file

Reply:
Done. I also extracted the variable for too small pies to the constant file. The new names are SMALL_BAR_THRESHOLD and SMALL_PIE_THRESHOLD.